index.js

```diff
@@ -0,0 +1,51 @@
+class Person {
+    constructor (name, dateOfBirth, money) {
```

Comment:
```suggestion
    constructor(name, dateOfBirth, money) {
```
In style guide no space before `(`. It is not critical, because you write all code in one style